StRoot/StEmcRawMaker/StBemcRaw.h

```diff
@@ -102,6 +102,7 @@ class StBemcRaw : public TObject
     controlADCtoE_st*        mControlADCtoE;
 
     Bool_t                   mSaveAllStEvent;
+    Bool_t                   mSaveAllBTOW;
```

Comment:
The layout of the data members of this class has changed, which implies that the ClassDef revision on line 260 should be incremented...
ClassDef(StBemcRaw, 2)

Alternatively, one can flag this value as non-persistent by using the "//!" comment at the end of the line.

----

Reply:
Hi Jason, this is a good catch! It doesn't hurt to do so. But I don't think StBemcRaw is saved into any file stream or be part of any class that is supposed to be saved in a file stream. It should be ClassDef(StBemcRaw, 0)